verifier/src/identity.rs

```diff
@@ -161,3 +179,120 @@ impl From<TrustedMrSignerIdentity> for TrustedIdentity {
         Self::MrSigner(mr_signer)
     }
 }
+
+// Convert two separate lists of config and sw hardening advisories into [`Advisories`]
+fn mitigated_advisories_to_advisories(
+    config_advisories: &[String],
+    sw_hardening_advisories: &[String],
+) -> Advisories {
+    let config_needed = config_advisories.is_empty().not();
```

Comment:
First time I'm seeing `.not()` instead of the `!` operator, but it makes sense that it works.

Reply:
I don't like how it's at the end, but the `!` is easy to gloss over at times